translations/en-us.yaml

```diff
@@ -2245,6 +2245,39 @@ clusterNew:
       placeholder: Your Client Secret
     region:
       label: Region
+    aadClientAppId:
+      label: Client ID
+      placeholder: Your Client ID of Azure Active Directory.
```

Comment:
Remove the periods in all the new placeholders, some have them and some don't. We should be consistent and for place holders we typically don't use periods.